docs/setup/existing-cluster.md

```diff
@@ -169,6 +169,31 @@ This is ideal for development and quick test environments.
          Complete
    ```
    
+#### 2. Selecting ingress mode
+User should select the suitable ingress mode: NodePort or Load balancer, based on their runtime(kubeadm, minikube, gcp, docker for desktop).
+
+```
+    ✔ Create
+    ✔ Existing cluster
+    ✔ Non persistent volume
+    ✔ Basic
+    [Use arrow keys]
+    ? Select ingress mode
+      ➤ Node port [kubeadm, minikube]
+        Load balancer [gcp, docker for desktop]
+        BACK
+```
+##### 2.1. NodePort
+If the ingress mode is NodePort user can either use a default NodePort ip address or provide a custom NodePort ip address.
+```
+    ✔ Create
+    ✔ Existing cluster
+    ✔ Non persistent volume
+    ✔ Basic
+    ✔ Node port [kubeadm, minikube]
+    ? NodePort Ip address:  [Press enter to use default NodePort ip address]
+```
+   
 ### Inline Method
```

Comment:
Please add the respective command to be passed for the inline method as well

Reply:
Updated.